src/gatsby-node.js

```diff
@@ -122,10 +218,10 @@ const fetchData = async (url, options = {}) => {
 }
 
 //strips special characters and makes string camelcase
-const customFormat = str => {
+const customFormat = (str) => {
   return str
     .replace(/^.*\/\/[^\/]+/, '') //Removes domain
-    .replace(/(?:^\w|[A-Z]|\b\w)/g, word => word.toUpperCase()) //Capitalizes strings
+    .replace(/(?:^\w|[A-Z]|\b\w)/g, (word) => word.toUpperCase()) //Capitalizes strings
```

Comment:
We probably want to touch upstream code as little as possible, to minimize the number of conflicts that need to be manually resolved whenever we merge from upstream. Certain things must be changed (which is why we have a fork in the first place) but anything we can get away with leaving alone, we should.